crates/sozo/ops/src/keystore.rs

```diff
@@ -85,10 +86,21 @@ pub fn inspect_private(password: Option<String>, raw: bool, file: PathBuf) -> Re
     Ok(())
 }
 
-fn get_password(password: Option<String>) -> std::io::Result<String> {
+fn get_password(password: Option<String>, retry: bool) -> std::io::Result<String> {
     if let Some(password) = password {
         Ok(password)
     } else {
-        rpassword::prompt_password("Enter password: ")
+        let password = rpassword::prompt_password("Enter password: ");
```

Comment:
```suggestion
        let password = rpassword::prompt_password("Enter password: ")?;
```
Let's propagate here to already have a string.